src/api/catchem.F90

```diff
@@ -0,0 +1,39 @@
+!> \file
+!! CATChem core data types and routines.
```

Comment:
should add a little more here for the grouping documentation 

`\defgroup API_CATChem`